### ISSUE TITLE
no such file

### ISSUE THREAD

**Issue by zllljf** [miao0913/SaCNN-CrowdCounting-Tencent_Youtu#12]
hello
when i run sh train_sacnn.sh
there is a bug:
`sed: can't read train_c.prototxt: No such file or directory
sed: can't read train_c.prototxt: No such file or directory
sed: can't read train_c.prototxt: No such file or directory
sed: can't read train_c.prototxt: No such file or directory
`
and i read the train_sacnn.sh + test_sacnn.sh, i find some files are not contained. 
i can not find the files called in train_sacnn.sh and test_sacnn.sh
In train_sacnn.sh:
1、how can i get the Train_Data and Train_GT
2、how can i get the log.txt
3、how can i download the pretrain_iter_95000.caffemodel. i can just download ShanghaiTech with PartA and PartB which contain ShanghaiTech_part_A.caffemodel and ShanghaiTech_part_B.caffemodel respective. 
Is pretrain_iter_95000.caffemodel the same as ShanghaiTech_part_A.caffemodel？

In test_sacnn.sh:
1、what is LIB stand for
2、how can i get lmdb
3、how can i get test_data/test_img
4、how can i get list.txt
5、how can i get estdmap.txt and estdmap.db

looking forward to your reply, thanks

**Comment by miao0913** [comment]
Yes it was implemented for GPU.

**Comment by zllljf** [comment]
There is a passage in your paper:
Because the head count regression is an easy task, we first regress the model on the density map (2); once it converges, we add (3) in the objective loss to jointly train a multi-task network for a few more epochs.
now i have trained a converges model just on density map, and want to add loss_count with trainrcl.prototxt, But i find the parameter of WEIGHT in solver.prototxt has NO EFFECT!!!!
so could please tell me ,how could i realize add (3) in the objective loss to jointly train a multi-task network for a few more epochs.
thanks
